server/src/weaverbird/backends/mongo_translator/steps/rename.py

```diff
@@ -0,0 +1,10 @@
+from typing import List
+
+from weaverbird.pipeline.steps import RenameStep
+
+
+def translate_rename(step: RenameStep) -> List:
+    return [
+        {'$addFields': {renaming[1]: f'${renaming[0]}'} for renaming in step.to_rename},
+        {'$project': {renaming[0]: 0} for renaming in step.to_rename},
```

Comment:
Same note as above, with a twist: if we want to rename column A to B and column B to C, then we'll end up with B being removed (only C in the output).
Even if this is a subtle edge case that should honestly not happen a lot, we can manage it quite easily iof we add a condition to the suppression: 
`for renaming in step.to_rename if renaming[1] not in new_column_names` with `new_column_names` being `[renaming[1] for renaming in setp.to_rename]`